src/industry_gui.cpp

```diff
@@ -1838,6 +1842,22 @@ class IndustryDirectoryWindow : public Window {
 				break;
 		}
 	}
+
+	EventState OnHotkey(int hotkey) override{
+		switch (hotkey){
```

Comment:
```suggestion
	EventState OnHotkey(int hotkey) override
	{
		switch (hotkey) {
```
Newline before `{` of function, space before other `{`s.